validation-test/stdlib/UnsafeBufferPointer.swift.gyb

```diff
@@ -105,6 +185,27 @@ ${SelfName}TestSuite.test("badNilCount")
   _ = buffer
 }
 
+${SelfName}TestSuite.test("subscript/get").forEach(in: subscriptGetTests) { 
+  (test) in
+  if test.expectedValues == nil {
+    expectCrashLater()
+  }
+  let elementCount = SubscriptGetTest.elementCount
+
+  var memory = UnsafeMutablePointer<OpaqueValue<Int>>(allocatingCapacity: elementCount)
+  let buffer = test.create${SelfName}(from: memory)
+  defer {
+    memory.deallocateCapacity(elementCount)
+  }
+
+  let slice = buffer[test.range]
```

Comment:
Maybe it's just me, but I would move the `expectCrashLater` immediately above this line. To emphasize which particular call is going to crash.